doc/langref.html.in

```diff
@@ -777,6 +777,30 @@
       implementation feature, not a language semantic, so it is not guaranteed to be observable to code.
       </p>
       {#header_close#}
+
+      {#header_open|Destructuring#}
+      <p>
+        Some aggregate values (e.g. {#link|Tuples#}, {#link|Arrays#}, {#link|Vectors#}) can be destructed using
```

Comment:
Is there anything else that can be destructed besides these 3? I would either remove "e.g." if not or add the last one and just make it complete

Reply:
corrected